src/operator/elemwise_op_common.h

```diff
@@ -73,6 +73,12 @@ inline bool ElemwiseStorageAttr(const nnvm::NodeAttrs& attrs,
     dispatched = storage_type_assign(out_attrs, kCSRStorage,
                                      dispatch_mode, dispatch_ex);
   }
```

Comment:
As usual, please update the documentation for the sparse operators during registration

Reply:
As usual, please add unit tests

Reply:
I think I mentioned in the comments part that there were already some unit tests written for this, please check that part out.

Reply:
Change for doc done.